### DISCUSSION TITLE
Linking snippets

### DISCUSSION THREAD

**Discussion by ChristophMarketsmueller** [pieces-app/support#178] [Ideas]
Hello together,
I think it would be helpful, if you could link snippets with other snippets similar to files and folders. 

For example, I have various HTML / SCSS and Javascript snippets that belong "to each other".

 It would be great if I could jump directly from one snippet to another snippet.

**Comment by joshuadanpeterson** [comment]
I use the tags for this, but having a dedicated feature would be cool

**Reply by joshuadanpeterson** [reply]
Frankie Bukenya suggested something similar in a recent "[Frontend Developer User Story](https://youtu.be/A4ggNc-sr8U?si=MT-8HAzDZAvmU-fM)" at 31:42.

**Reply by ChristophMarketsmueller** [reply]
Yes this is exactly what i meant :)

**Reply by joshuadanpeterson** [reply]
Cool, hopefully they implement it. It'd be a great addition!

**Comment by Raumo0** [comment]
Apart from the aforementioned use case, it is currently very difficult to capture patterns from multiple files, as well as project structures that require a hierarchy of snippets. I do think it is very important to implement this for the full use of pieces.
